regression/rrfs_3denvar_rdasens.sh

```diff
@@ -68,19 +64,59 @@ fixcrtm=${fixcrtm:-$CRTM_FIX}
 #
 #---------------------------------------------------------------------
 #
-echo "regional_ensemble_option is ",${regional_ensemble_option:-1}
-
+regional_ensemble_option=${regional_ensemble_option:-5}
+NUM_ENS_MEMBERS=5
+echo "regional_ensemble_option is ",${regional_ensemble_option}
+echo "regional_ensemble number is ",${NUM_ENS_MEMBERS}
 echo "$VERBOSE" "fixgsi is $fixgsi"
-echo "$VERBOSE" "fixgriddir is $fixgriddir"
 echo "$VERBOSE" "default bkpath is $bkpath"
 echo "$VERBOSE" "background type is is $BKTYPE"
 
 ifhyb=.false.
-if  [[ ${regional_ensemble_option:-1} -eq 1 ]]; then #using GDAS
+#
+# Check if we have enough FV3-LAM ensembles when regional_ensemble_option=5
+#
+if  [[ ${regional_ensemble_option} -eq 5 ]]; then
```

Comment:
a little confusing to me.  Will this ctest be supposed to test using either  regional_ensemble_option==5 or ==1 at the user's choice?  That means a much complicated while doable ctest (maybe just to split to two different ctests is better).  If not the case, only ==5 is to be used.  Could the logics here be simplified ?

Reply:
@TingLei-NOAA You are right that there are other possible configuration. But this case is to match the RRFS operational configuration. So I set regional_ensemble_option=5 but leave the check there.